src/getting-started.md

```diff
@@ -259,9 +259,9 @@ Eventually, the Translation Proxy log output will show sucessful share, which me
 ```
 
 
-## Config D: SV1 firmware > Translation Proxy JN (Job Negotiator) > SV2 Pool
+## Config D: SV1 firmware > Translation Proxy JN (job declarator) > SV2 Pool
 
-This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **Job Negotiator (JN)** sub-protocol: allowing miners to select transactions locally and send them to the **Pool-side JN**. 
+This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **job declarator (JN)** sub-protocol: allowing miners to select transactions locally and send them to the **Pool-side JN**. 
```

Comment:
- to implement the `Job Declaration (JD)` sub-protocol
- them to the Pool-side `JDS (Job Declarator Server)`.